.github/CODEOWNERS

```diff
@@ -2539,7 +2539,6 @@
 /types/iost-contract/                                   @monaka
 /types/iota.lib.js/                                     @fogsh
 /types/ip/                                              @codeanimal @BendingBender
-/types/ip-address/                                      @icopp @danwbyrne
```

Comment:
IMO this is automatic, done after removal is done, please amned this to not introduce a change

Reply:
@peterblazejewicz fixed 👍